Gordon360/Models/CCT/LostAndFound/Missing.cs

```diff
@@ -1,4 +1,6 @@
-﻿using System;
+﻿// <auto-generated> This file has been auto generated by EF Core Power Tools. </auto-generated>
+#nullable disable
```

Comment:
We're not using this file anymore right?

Reply:
Correct, I believe no, since this is the old db table.  I'll remove it from this PR.  We'll also drop the db table once we push both frontend and backend changes.